R/dbConfig.R

```diff
@@ -1,15 +1,13 @@
 library(DBI)
```

Comment:
As you've moved this into a function in data.R, I suggest deleting this file - in general, I suggest deleting things you no longer need - it makes the codebase cleaner and easier to maintain / update. (You can always get the deleted things back by going into the git history if you need them!)